File: recyclerview-swipe/src/main/java/com/yanzhenjie/recyclerview/swipe/widget/GridItemDecoration.java
/*
 * AUTHOR：YanZhenjie
 *
 * DESCRIPTION：create the File, and add the content.
 *
 * Copyright © www.mamaqunaer.com. All Rights Reserved
 *
 */
package com.yanzhenjie.recyclerview.swipe.widget;

import android.support.annotation.ColorInt;

/**
 * Created by YanZhenjie on 2017/7/6.
 *
 * @deprecated use {@link DefaultItemDecoration} instead.
 */
@Deprecated
public class GridItemDecoration extends DefaultItemDecoration {

    public GridItemDecoration(@ColorInt int color) {
        super(color);
    }

    public GridItemDecoration(@ColorInt int color, int dividerWidth, int dividerHeight, int... excludeViewType) {
        super(color, dividerWidth, dividerHeight, excludeViewType);
    }
}

File: recyclerview-swipe/src/main/java/com/yanzhenjie/recyclerview/swipe/widget/GridItemDecoration.kt
/*
 * AUTHOR：YanZhenjie
 *
 * DESCRIPTION：create the File, and add the content.
 *
 * Copyright © www.mamaqunaer.com. All Rights Reserved
 *
 */
package com.yanzhenjie.recyclerview.swipe.widget

import android.support.annotation.ColorInt

/**
 * Grid item decoration.
 *
 * Created by YanZhenjie on 2017/7/6.
 */
@Deprecated("use {@link DefaultItemDecoration} instead.")
class GridItemDecoration : DefaultItemDecoration {

  constructor(@ColorInt color: Int) : super(color)

  constructor(@ColorInt color: Int, dividerWidth: Int, dividerHeight: Int, vararg excludeViewType: Int) : super(color, dividerWidth, dividerHeight, *excludeViewType)

}
